webserver/views/api.py

```diff
@@ -37,7 +38,13 @@ def get_low_level(mbid):
     try:
         return Response(load_low_level(mbid, offset), content_type='application/json')
     except NoDataFoundException:
-        raise NotFound
+        raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
+
+@api_bp.route("/<string:notanmbid>/low-level", methods=["GET"])
+@crossdomain()
+def get_low_level_404(notanmbid):
+    """Specific low-level view for non-mbid url parts to return a json error message"""
+    raise webserver.exceptions.InvalidAPIUsage("Not found", status_code=NotFound.code)
```

Comment:
Right, that's what I thought. I think just having a regular checking within one view would be better.